src/todo/AddOrEditList.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useCallback } from "react";
+import { connect } from "react-redux";
+import * as actions from "../store/actions/list";
+import {
+  Typography,
+  TextField,
+  FormLabel,
+  Container,
+  InputLabel,
+  FormControl,
+  Button,
+  Box,
+  makeStyles,
+} from "@material-ui/core";
+
+import Lists from "./Lists";
+import AddOrEditTodoInList from "./AddOrEditTodoInList";
+
+const useStyles = makeStyles((theme) => ({
+  inputList: {
+    marginTop: theme.spacing(2),
+    minWidth: 300,
+  },
+  btnAddList: {
+    marginTop: theme.spacing(2),
+    marginLeft: theme.spacing(1),
+  },
+  root:{
+    display: "flex",
+    flexDirection:'column',
+    [theme.breakpoints.up("sm")]: {
+      flexDirection:'row',
+     },
+   }
+}));
```

Comment:
Keep this style in separate file.
create a file in src/todo/style.ts

and declare this useStyle Object there you can rename this to AddOrEditListStyles and export it from there.

Reply:
done